packages/autorest.python/test/vanilla/legacy/Expected/AcceptanceTests/Constants/constants/models/_models_py3.py

```diff
@@ -34,10 +34,7 @@ class ModelAsStringNoRequiredOneValueDefault(_serialization.Model):
     }
 
     def __init__(
-        self,
-        *,
-        parameter: Optional[Union[str, "_models.ModelAsStringNoRequiredOneValueDefaultEnum"]] = None,
-        **kwargs: Any
+        self, *, parameter: Union[str, "_models.ModelAsStringNoRequiredOneValueDefaultEnum"] = "value1", **kwargs: Any
```

Comment:
It shall have default value since `x-ms-client-default` is defined: https://github.com/Azure/autorest.testserver/blob/b6195830209d783dcfef44803f4748f35c1241e0/swagger/constants.json#L581-L591